cmd/vfkit/main.go

```diff
@@ -110,18 +112,41 @@ func waitForVMState(vm *vz.VirtualMachine, state vz.VirtualMachineState) error {
 	}
 }
 
-func runVirtualMachine(vmConfig *config.VirtualMachine) error {
-	vzVMConfig, err := vf.ToVzVirtualMachineConfig(vmConfig)
+func runVFKit(vmConfig *config.VirtualMachine, opts *cmdline.Options) error {
+	uri, err := rest.ParseRestfulURI(opts.RestfulURI)
 	if err != nil {
 		return err
 	}
 
+	scheme, err := rest.ToRestScheme(uri.Scheme)
+	if err != nil {
+		return err
+	}
+
+	vzVMConfig, err := vf.ToVzVirtualMachineConfig(vmConfig)
+	if err != nil {
+		return err
+	}
 	vm, err := vz.NewVirtualMachine(vzVMConfig)
 	if err != nil {
 		return err
 	}
 
-	err = vm.Start()
+	// Do not enable the rests server if user sets scheme to None
+	if scheme != rest.None {
```

Comment:
I'd prefer to just have: 
```
    if opts.RestfulURI != "" { // or cmdline.DefaultRestURI instead of ""
        srv, err := rest.NewServer(..., opts.RestfulURI)
        // error handling
        go func() {
            if err := srv.Start(); err != nil {
				log.Error(err)
	    }
        }
````
This way we don't need to have the URL parsing (`ParseRestfulURI`, `ToRestScheme`) in `runVFKit`